package.json

```
@@ -49,6 +49,7 @@
 			"@swc/core",
 			"core-js",
 			"esbuild",
+			"print-this",
```

I also recently [added](https://github.com/Automattic/jetpack/pull/44685) `jquery-cycle` to Jetpack dependencies, should we add it here as well?

---

No, this is a set of packages that try to run unneeded post-install scripts, which we disallow here:
https://github.com/Automattic/jetpack/blob/931b1867d1046c5f02db271b0c471140e74b2522/.npmrc#L31-L32

Because we disallow post-install scripts, `pnpm` outputs an error, and the above suppresses that error.

---

For reference, [here](https://github.com/jasonday/printThis/blob/23be1f8ad7ab3b3526c43dc47775279edd65bd85/package.json#L12) is the post-install script it's trying to run.